kimchi/src/circuits/constraints.rs

```diff
@@ -52,14 +52,14 @@ pub struct FeatureFlags {
 /// The polynomials representing evaluated columns, in coefficient form.
 #[serde_as]
 #[derive(Clone, Serialize, Deserialize, Debug)]
-pub struct EvaluatedColumnCoefficients<F: PrimeField> {
+pub struct EvaluatedColumnCoefficients<const W: usize, F: PrimeField> {
```

Comment:
Nit: here (and elsewhere) it would be helpful to give a more descriptive name than `W`. `COLUMNS` (or `NUM_COLUMNS`, `WITNESS_SIZE`, etc.) would be more self-documenting.

Reply:
Oh sure! I was trying to keep it as succinct as possible (witness width), but I can for sure be more explicit 👍🏻